src/components/ContentItem/ContentItem.vue

```diff
@@ -2,6 +2,7 @@
   <li class="item"
       :data-testid="dataTestId"
       :disabled="disabled"
+      :class="{ success }"
```

Comment:
(nit) please move this to a new line above line: 3, so that it's closer to the `class` attribute